monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -379,7 +412,8 @@
             'HadoopExploiter': ReportService.process_hadoop_exploit,
             'MSSQLExploiter': ReportService.process_mssql_exploit,
             'VSFTPDExploiter': ReportService.process_vsftpd_exploit,
-            'DrupalExploiter': ReportService.process_drupal_exploit
+            'DrupalExploiter': ReportService.process_drupal_exploit,
+            'ZerologonExploiter': ReportService.process_zerologon_exploit
```

Comment:
This is a messy legacy code design with tons of duplication in these functions. We should refactor these sometime

Reply:
@VakarisZ async, can we use a short and boring plugin system on the island side?